src/components/tabs.js

```diff
@@ -0,0 +1,87 @@
+/*
+ This file is a part of libertysoil.org website
+ Copyright (C) 2016  Loki Education (Social Enterprise)
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import bem from '../utils/bemClassNames';
+
+export default class Tab extends React.Component {
+  render() {
+    const className = this.props.className + ' tabs__tab' + (this.props.active ? '' : ' hidden');
+
+    return (
+      <div className={className}>{this.props.children}</div>
+    );
+  }
+}
+
+export default class Tabs extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      active: 0
+    };
+  }
+
+  // static propTypes = {
+  //   children: PropTypes.oneOf([
+  //     PropTypes.array,
+  //     PropTypes.element
+  //   ])
+  // };
+
+  static defaultProps = {
+    active: 0
+  };
+
+  componentWillMount() {
+    this.setState({ active: this.props.active });
+  }
+  
+  clickHandler = (i) => {
+    this.setState({ active: i });
+  };
+
+  render() {
+    const titles = React.Children.map(this.props.children, function(item, i) {
+      const className = bem.makeClassName({
+        block: 'tabs__title',
+        modifiers: {
+          active: i === this.state.active
+        }
+      });
+      return (<li className={className} onClick={this.clickHandler.bind(null, i)} key={i}>{item.props.title}</li>);
+    }, this);
+
+    const tabs = this.props.children.map(function(item, i) {
```

Comment:
I use additional argument of `map`. However, is it required to rewrite the function to "arrow"?